mobile/lib/features/trade/settings_screen.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:flutter/material.dart';
+import 'package:get_10101/features/trade/trade_screen.dart';
+
+class TradeSettingsScreen extends StatelessWidget {
```

Comment:
If we split the settings screens between `TradeSetting` and `Wallet Settings`, what can you configure in either?

Reply:
This split is indeed a bit tricky - and is not really what we want.

I added another commit that adds a common settings screen that is used in both routes; this allows us to have the same settings screen for both wallet and trade (including general settings and app info), but potentially highlight the settings that are relevant when opening the screen:

![2023-02-06 14 53 54](https://user-images.githubusercontent.com/5557790/216880267-9aff96d9-6218-4589-b9dc-5117ce96e7d2.gif)

Note: This is not totally in line with "each screen should be identified by a route"; but I did not find a better solution for now.